pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,102 @@
+
+[tip type="important"]
+ Bento is only available [experimentally](../learn/experimental.md). Read our [Blog Post](TODO) for information on future plans and how to get involved! 
+[/tip]
+
+# Use Bento components in non-AMP pages
+
+Bento components are available for experimental use in the developer preview.
+
+
+## Enable Bento Experiment 
+
+Bento components are available experimentally and you must [enable experimental features](../learn/experimental.md).
+
+You can enable all Bento components by including the script below:
+
+
+```
+<script>
+  (self.AMP = self.AMP || []).push(function (AMP) {
+    AMP.toggleExperiment('bento', true);
+  });
+</script>
+```
+
+
+Or enable a single Bento component by including the script below. Ensure you replace `bento-foo` with the desired [supported component](#available-bento-components) name given on its documentation page:
+
+
+```
+<script>
+  (self.AMP = self.AMP || []).push(function (AMP) {
+    AMP.toggleExperiment('bento-foo', true);
+  });
+</script>
+```
+
+
+
+## Import AMP runtime and Bento component logic
+
+You must include the AMP runtime script and import the logic for each individual Bento component desired.
+
+```
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+
+<script async custom-element="amp-bento" src="https://cdn.ampproject.org/v0/amp-bento-1.0.js"></script>
+``` 
+
+Read each Bento component’s reference documentation for implementation details.
+
+
+# Available Bento components 
+
+Bento supported AMP components are listed below:
+
+
+
+*   [amp-accordion](../../../documentation/components/reference/amp-accordion.md)
+*   [amp-base-carousel](../../../documentation/components/reference/amp-base-carousel.md)
+*   [amp-inline-gallery](../../../documentation/components/reference/amp-inline-gallery.md)
+    *   amp-inline-gallery-pagination
+    *   amp-inline-gallery-thumbnails
+*   [amp-stream-gallery](TODO)
+*   [amp-date-countdown](../../../documentation/components/reference/amp-date-countdown.md)
+*   [amp-date-display](../../../documentation/components/reference/amp-date-display.md)
+*   [amp-fit-text](../../../documentation/components/reference/amp-fit-text.md)
+*   [amp-instagram](../../../documentation/components/reference/amp-instagram.md)
+*   [amp-lightbox](../../../documentation/components/reference/amp-lightbox.md)
+*   [amp-selector](../../../documentation/components/reference/amp-selector.md)
+*   [amp-social-share](../../../documentation/components/reference/amp-social-share.md)
+*   [amp-timeago](../../../documentation/components/reference/amp-timeago.md)
+*   Video components
+    *   [amp-video](../../../documentation/components/reference/amp-video.md)
+    *   [amp-youtube](../../../documentation/components/reference/amp-youtube.md)
+    *   [amp-video-iframe](../../../documentation/components/reference/amp-video-iframe.md)
```

Comment:
Please remove, I don't see a `1.0` registered in AMPHTML for `amp-video-iframe`.